client/blocks/jitm/templates/sidebar-banner.jsx

```diff
@@ -1,15 +1,57 @@
 /**
- * External Dependencies
+ * External dependencies
  */
 import React from 'react';
 import Gridicon from 'components/gridicon';
+import { get } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import UpsellNudge from 'blocks/upsell-nudge';
+import { abtest } from 'lib/abtest';
 
 /**
  * Style dependencies
  */
 import './sidebar-banner.scss';
 
-export default function SidebarBannerTemplate( { icon, CTA, message, onClick, trackImpression } ) {
+export default function SidebarBannerTemplate( {
+	icon,
+	CTA,
+	message,
+	onClick,
+	trackImpression,
+	id,
+	event,
+	featureClass,
+	tracks,
+	...props
+} ) {
+	if ( abtest( 'sidebarUpsellNudgeUnification' ) === 'variantShowUnifiedUpsells' ) {
+		const clickName = get( tracks, [ 'click', 'name' ] ) || `jitm_nudge_click_${ id }`;
```

Comment:
The Calypso project now supports [Optional Chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining)  and [nullish coalesing](https://github.com/tc39/proposal-nullish-coalescing/) So it's possible to remove the lodash`get` import in favor of:

```jsx
const clickName = tracks?.click?.name ?? `jitm_nudge_click_${ id }`;
const clickProps = tracks?.click?.props;
const displayName = tracks?.display?.name ?? `jitm_nudge_impression_${ id }`;
```

Reply:
TIL!